package-lock.json

```diff
@@ -0,0 +1,10778 @@
+{
```

Comment:
Hey @0xTimepunk, thanks for the PR!

This file isn't needed. We use `yarn` on the monorepo and the lock file sits at the root.